site/content/docs/user/known-issues.md

```diff
@@ -306,6 +313,34 @@ your workloads inside the cluster via the nodes IPv6 addresses.
 
 See Previous Discussion: [kind#1326]
 
+## Failed to get rootfs info / "stat failed on /dev/..."
+
+On some systems, creating a cluster times out with these errors in kubelet.log (device varies):
+
+```
+stat failed on /dev/nvme0n1p3 with error: no such file or directory
+"Failed to start ContainerManager" err="failed to get rootfs info: failed to get device for dir \"/var/lib/kubelet\": could not find device with major: 0, minor: 40 in cached partitions map"
+```
+
+Kubernetes needs access to storage device nodes in order to do some stuff, e.g. tracking free disk space. Therefore, Kind needs to mount the necessary device nodes from the host into the control-plane container — however, it cannot always determine which device Kubernetes requires, since this varies with the host filesystem. For example, Kind doesn't handle BTRFS, which is the default for modern Fedora.
+
+This can be worked around by including the necessary device as an extra mount in the cluster configuration file.
```

Comment:
I guess, setting `$KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER` to `native` or `fuse-overlafys` may work too?

Reply:
That I don't know, and can't test right now. Is that documented somewhere? I've been using Kind for about 2 days now, so not an expert by any means.

Reply:
Ok, did some quick experimenting today. Assuming you're expecting something like:
```sh
KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER=native kind create cluster --retain
```

...then neither `native` nor `fuse-overlayfs` has any apparent effect. Both fail with the original problem, "stat failed on /dev/nvme0n1p3 with error".